ivadomed/loader/adaptative.py

```diff
@@ -482,8 +472,9 @@ class HDF5Dataset:
     """HDF5 dataset object.
 
     Args:
+        bids_df (BidsDataframe): Object containing dataframe with all BIDS image files and their metadata.
         path_data (list) or (str): Path(s) to BIDS datasets
-        subject_lst (list of str): List of subjects.
+        subject_lst (list of str): List of subject filenames.
```

Comment:
might be better to rename `subject_lst` to `subject_file_lst` for code cohesion

Reply:
Good catch, thanks!
I fixed it in 4e7ce79.
I made similar changes to split functions names for code cohesion in 991c3b6.